pkg/meta/interface.go

```diff
@@ -45,13 +45,13 @@ const (
 )
 
 const (
-	TypeFile      = 1 // type for regular file
-	TypeDirectory = 2 // type for directory
```

Comment:
The number is persisted into database, we should NOT change any of it, so we'd prefer to use the number here.

It's also more readable to people who is not familiar with go.